blocks/footer/footer.js

```diff
@@ -15,5 +16,6 @@ export default async function decorate(block) {
   const footer = document.createElement('div');
   footer.innerHTML = html;
   await decorateIcons(footer);
+  wrapImgsInLinks(footer);
```

Comment:
@cuzuco2 you only need the above, if a "static" image (like the logo in the header) should receive a link (like the home link on logo in the header). 

As far as i can see, the footer contains no images.

Please remove

Reply:
Ok I will, I was thinking that sometimes this is the kind of thing that we see for some customers and did hesitate to add it or not, and I decided to let it in case the customer later wants to achieve something similar.
But if we block this and don't include any section that will allow the customer to later add such a thing then yes it doesn't make sense.
I will remove this

Reply:
i understand. unless there's a concrete use case, we should follow the minimalsim principle. anything extraneous that is not needed has the potential to influence the lighthouse score negatively.